code/modules/reagents/reagent_containers/bottle.dm

```diff
@@ -40,7 +43,7 @@
 				filling.icon_state = "[icon_state]100"
 
 		filling.icon += mix_color_from_reagents(reagents.reagent_list)
-		overlays += filling
+		underlays += filling
```

Comment:
It would honestly be better to shade the filling differently to have that effect rather than to uh, well...utilize underlays.